### PR TITLE
crc: 2.6.0 -> 2.11.0

### PR DESCRIPTION
**Pull request by kadel**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by tricktron — CHANGES_REQUESTED** [review]

**Review by tricktron — APPROVED** [review]
Builds & works on `aarch64-darwin`👍.

**Comment by tricktron** [comment]
@GrahamcOfBorg build crc crc.passthru.tests.version

**Comment by SuperSandro2000** [comment]
Please rebase

**Review by tricktron — APPROVED** [review]
LGTM

File: pkgs/applications/networking/cluster/crc/default.nix [hunk 1]
```diff
@@ -40,22 +43,20 @@ buildGoModule rec {
 
   tags = [ "containers_image_openpgp" ];
 
+
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/crc/default.nix:46]:
```suggestion
```

File: pkgs/applications/networking/cluster/crc/default.nix [hunk 2]
```diff
@@ -10,20 +10,23 @@
...
   writeKey = "cvpHsNcmGCJqVzf6YxrSnVlwFSAZaYtp";
 in
 buildGoModule rec {
-  version = "2.6.0";
+  version = "2.11.0";
   pname = "crc";
-  gitCommit = "6b954d40ec3280ca63e825805503d4414a3ff55b";
+  gitCommit = "a5f90a25abcacd4aa334490f0d204329abeaa691";
+  modRoot = "cmd/crc";
+
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/cluster/crc/default.nix:23]:
```suggestion
```

**Pull request merged**